.github/workflows/upload_pypi.yml

```diff
@@ -11,8 +11,8 @@ jobs:
       max-parallel: 4
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
-        platform: [macos-latest, windows-latest]  # No wheels on linux yet
+        python-version: [3.7, 3.8, 3.9, 3.10]
```

Comment:
Should this 3.10 be '3.10' like the others? I see it's in a '.yml' file, so maybe it's smart enough, but if these get parsed as a python list, it might end up as 3.1 rather than '3.10'.

Reply:
Weird -- I had already changed that...not sure why the quotes aren't showing up for you.

Reply:
Do you see the quotes there now? Line 14 of .github/workflows/upload_pypi.yml ?

Reply:
Indeed! I see them in your branch, but not in the PR for some reason. Weird.